rusk-uplink/Cargo.toml

```diff
@@ -7,6 +7,6 @@ edition = "2018"
 
 [dependencies]
 microkelvin = { version = "0.16.0-rkyv", default-features = false }
-rkyv = { version = "0.7.26", default-features = false, features = ["alloc", "validation"] }
+rkyv = { version = "0.7.29", default-features = false, features = ["alloc", "validation"] }
```

Comment:
Is there a reason we want to specify the minor version? I care because this is likely to become a public crate.

Reply:
here I'd be rather careful, I think we don't want to progress with rkyv versions without losing control at the moment, at least not at this stage yet, I'd keep this thought for the future though

Reply:
Alright. Lets add a comment to revisit the subject at a later date.